mail/views_test.py

```diff
@@ -252,6 +252,72 @@ def test_course_team_email_unpaid(self):
         assert resp.status_code == status.HTTP_403_FORBIDDEN
 
 
+class LearnerMailViewTests(APITestCase, MockedESTestCase):
```

Comment:
Can you add a test where a user who is not staff attempts to use this API to send an email?